Tests/scripts/hook_validations/structure.py

```diff
@@ -45,7 +45,8 @@ class StructureValidator(object):
         SCRIPT_JS_REGEX,
         INTEGRATION_JS_REGEX,
         INTEGRATION_PY_REGEX,
-        REPUTATION_REGEX
+        REPUTATION_REGEX,
+        BETA_INTEGRATION_YML_REGEX
```

Comment:
this will fail the master.
some beta files works but do no pass schema. since it is beta and there is no instance(mostly) we cannot promised we can fix the yml files because we can't test.